rpc/sigpipe.go

```diff
@@ -0,0 +1,9 @@
+// +build !darwin
+
+package rpc
+
+import "net"
+
+func DisableSigPipe(c net.Conn) error {
+	return nil
```

Comment:
If you really want an option for future-proofing, maybe you can invert it to something like `EnableDarwinSigPipe`, which would be false by default, in case we ever have a situation where the caller needs to ensure it's enabled?